.github/workflows/ci.yml

```diff
@@ -303,13 +308,13 @@ jobs:
     steps:
       - name: Generate token
         id: generate_token
-        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
+        uses: actions/create-github-app-token@3ff1caaa28b64c9cc276ce0a02e2ff584f3900c5 # v2.0.2
         with:
-          app_id: ${{ secrets.PLUGINS_PLATFORM_BOT_APP_ID }}
-          private_key: ${{ secrets.PLUGINS_PLATFORM_BOT_APP_PEM }}
+          app-id: ${{ secrets.PLUGINS_PLATFORM_BOT_APP_ID }}
```

Comment:
@jackw we should move this to use vault, the secrets here are not longer valid. (we should delete them too)